src/bookshelf/Makefile

```diff
@@ -25,6 +25,9 @@ all: $(REBAR3)
 rel: all
 	@$(REBAR3) release
 
+update: $(REBAR3)
```

Comment:
Were these  pulled in from concrete upstream?   If not, it'll be overwritten on the next concrete update (same for the other Makefile changes)